docs/sources/enterprise/reporting.md

```diff
@@ -55,18 +90,37 @@ The last saved version of the report will be sent to selected emails. You can us
 
 {{< docs-imagebox img="/img/docs/enterprise/reports_send_test_mail.png" max-width="500px" class="docs-image--no-shadow" >}}
 
+## Send report via the API
+
+Sending reports programmatically is possible with the [send report]({{< relref "http_api.md#send-report" >}}) endpoint in the [Enterprise HTTP APIs]({{< relref "http_api.md" >}}).
```

Comment:
```suggestion
You can send reports programmaticallywith the [Reporting API]({{< relref "http_api.md#send-report" >}}) in the [Enterprise HTTP APIs]({{< relref "http_api.md" >}}).
```

Reply:
I'd like to refer to the exact endpoint, not the API as a whole, made the change to active voice but kept the endpoint reference.